### PR TITLE
api fetch and card styling

### PR DESCRIPTION
**Pull request by melfuechec**
-add getMovies and getMovieImg api requests
-create style for MovieDescription component

### CODE REVIEW

File: src/MovieDescription.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from "react";
+ import getMovieImg from './utils/getMovieImg'
```
Comment (mfuechec) [inline src/MovieDescription.js:2]: I think we may need to add a linter to this repo in the future.

File: src/style/MovieDescription.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.card
+ {
+ border: 1px solid black;
+ height: 300px;
```
Comment (mfuechec) [inline src/style/MovieDescription.css]: I don't think this file is being used.

File: src/utils/getMovieImg.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const baseURL = 'https://image.tmdb.org/t/p/w500';
+
+const getMovieImg=(setimgURL, filePath) => {
+    fetch(baseURL+filePath)
+    .then((data) => { setimgURL(data.url) })
```
Comment (mfuechec) [inline src/utils/getMovieImg.js]: These curly braces around setImgURL are not necessary.

File: src/utils/getMovies.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { key } from "../secrets"
+const api = `https://api.themoviedb.org/3/movie/upcoming?api_key=${key}`
+
+
+const getMovies = (setMovies) => {
+    fetch(api)
+  .then((response) => response.json())
+  .then((data) => { setMovies(data) });
```
Comment (mfuechec) [inline src/utils/getMovies.js]: Again, curly braces unnecessary.

**Review by mfuechec — CHANGES_REQUESTED** [review]
Looks good, but could you add a description along with this pull request.

File: src/MovieDescription.js (hunk 1 above)
Comment (melfuechec) [reply src/MovieDescription.js:2]: so smort!

**Review by mfuechec — APPROVED** [review]

**Pull request merged**